packages/fxa-settings/src/lib/integrations/integration-factory-flags.test.ts

```diff
@@ -5,12 +5,12 @@
 import { createSandbox, SinonSandbox } from 'sinon';
 import { Constants } from '../constants';
 import { StorageData, UrlQueryData } from '../model-data';
-import { DefaultRelierFlags } from './relier-factory-flags';
+import { DefaultIntegrationFlags } from './integration-factory-flags';
 import { ReachRouterWindow } from '../window';
 
-describe('lib/reliers/relier-factory-flags', function () {
+describe('lib/integrations/relier-factory-flags', function () {
```

Comment:
Q: is this `relier-factory-flags` or `integration-factory-flags`?

Reply:
Thanks, should be `integrations/integration-factory-flags`; hopefully I can catch all the stray references lol.

Reply:
<kbd>grep</kbd> to the rescue!

```sh
git grep "relier-factory" | cat

packages/fxa-settings/src/lib/integrations/integration-factory-flags.test.ts:describe('lib/integrations/relier-factory-flags', function () {
```

I'll have to look through the PR again to see if other files were changed that we might want to search for, but it's also just a test name, so... whhhaaattteevvveeerrrrrr.